ush/python/pygfs/task/land_analysis.py

```diff
@@ -183,7 +183,10 @@ def initialize(self) -> None:
         FileHandler({'mkdir': dirlist}).sync()
 
         # stage fix files
-        jedi_fix_list_path = os.path.join(self.task_config.HOMEgfs, 'parm', 'parm_gdas', 'land_jedi_fix.yaml')
+        if localconf.FRAC_GRID:
+            jedi_fix_list_path = os.path.join(self.task_config.HOMEgfs, 'parm', 'parm_gdas', 'land_jedi_fix_gfsv17.yaml')
+        else:
+            jedi_fix_list_path = os.path.join(self.task_config.HOMEgfs, 'parm', 'parm_gdas', 'land_jedi_fix.yaml')
```

Comment:
I think we can do this in the `land_jedi_fix.yaml` itself instead of in the python script.